lib/slack/web/faraday/connection.rb

```diff
@@ -11,7 +11,7 @@ def connection
 
           options[:headers]['User-Agent'] = user_agent if user_agent
           options[:proxy] = proxy if proxy
-          options[:ssl] = { ca_path: ca_path, ca_file: ca_file }
+          options[:ssl] = { ca_path: ca_path, ca_file: ca_file } unless ca_path.nil? || ca_file.nil?
```

Comment:
This could be tricky.

Do both options have to be specified? Can't just one do the trick? I think one certificate chain can work just fine as well as just the cert directory.

So this could be nicer as `if ca_path || ca_file`. Unless with `||` can be evil.

Reply:
agree. if ca_path || ca_file is better. And only one needs to be present.